app/components/CrashRecommendationCard.tsx

```diff
@@ -0,0 +1,297 @@
+        addPartners.push({
+          partner_id: partnerNew.partner_id,
+          ...(recommendationId && {
+            recommendation_id: recommendationId,
+          }),
+        });
+      }
+    });
+  }
+  return [addPartners, deletePartnerPks];
+};
+
+interface CrashRecommendationCardProps {
+  recommendation: Recommendation | null;
+  crash_pk: number;
+  onSaveCallback: () => Promise<void>;
+}
+
+/**
+ * UI component for viewing and editing fatalitiy review
+ * board (FRB) recommendations
+ */
+export default function CrashRecommendationCard({
+  recommendation,
+  crash_pk,
+  onSaveCallback,
+}: CrashRecommendationCardProps) {
+  const { user } = useAuth0();
+  const [isEditing, setIsEditing] = useState(false);
+  const [isMutating, setIsMutating] = useState(false);
+  const { data: statuses, isLoading: isLoadingStatuses } =
+    useQuery<RecommendationStatus>({
+      query: isEditing ? RECOMMENDATION_STATUS_QUERY : null,
+      typename: "statuses",
+    });
+
+  const { data: partners, isLoading: isLoadingPartners } =
+    useQuery<CoordinationPartner>({
+      query: isEditing ? RECOMMENDATION_PARTNERS_QUERY : null,
+      typename: "partners",
+    });
+
+  const {
+    register,
+    reset,
+    handleSubmit,
+    formState: { isDirty },
+    setValue,
+    watch,
+  } = useForm<RecommendationFormInputs>({
+    defaultValues: recommendation
+      ? {
+          rec_text: recommendation.rec_text,
+          rec_update: recommendation.rec_update,
+          recommendation_status_id: recommendation.recommendation_status_id,
+          crash_pk: recommendation.crash_pk,
+          recommendations_partners:
+            recommendation.recommendations_partners || [],
+        }
+      : {
+          rec_text: null,
+          rec_update: null,
+          recommendation_status_id: null,
+          crash_pk,
+          created_by: user?.email,
+          recommendations_partners: [],
+        },
+  });
+
+  const { mutate } = useMutation(
+    recommendation
+      ? UPDATE_RECOMMENDATION_MUTATION
+      : INSERT_RECOMMENDATION_MUTATION
+  );
+
+  const onSave = useCallback(
+    async (data: RecommendationFormInputs) => {
+      setIsMutating(true);
+      // just use a generic object to make the data structuring easier
+      const payload: Record<string, unknown> = data;
+
+      const [addPartners, deletePartnerPks] = getPartnerChanges(
+        recommendation?.id || null,
+        recommendation?.recommendations_partners || [],
+        data.recommendations_partners || []
+      );
+
+      let variables;
+      if (recommendation) {
+        delete payload.recommendations_partners;
+        variables = {
+          record: payload,
+          id: recommendation?.id,
+          addPartners,
+          deletePartnerPks,
+        };
+      } else {
+        payload.recommendations_partners = { data: addPartners };
+        variables = { record: payload };
+      }
+      await mutate(variables, { skip_updated_by_setter: true });
```

Comment:
cool, i haven't seen this mutate option to skip setting who updated a record in the code yet. It is nice that it defaults to capture this data in the mutation. So, then we'll just remove this when `updated_by` is added to recommendations table in the future. 👍